src/AdoNetCore.AseClient/Token/ClientCapabilityToken.cs

```diff
@@ -0,0 +1,462 @@
+                //cap request
+                ((byte)Byte0.Unknown),
+                ((byte)Byte1.Unknown),
+                ((byte)(/*Byte2.REQ_UNUSED1 | Byte2.REQ_UNUSED2 | Byte2.REQ_UNUSED3 | Byte2.REQ_UNUSED4 | Byte2.REQ_UNUSED5 | Byte2.REQ_COMMAND_ENCRYPTION | Byte2.REQ_READONLY |*/ Byte2.REQ_DYNAMIC_SUPPRESS_PARAMFMT)),
+                ((byte)(Byte3.REQ_LOGPARAMS | Byte3.REQ_ROWCOUNT_FOR_SELECT | Byte3.DATA_LOBLOCATOR/*| Byte3.REQ_RPC_BATCH*/| Byte3.REQ_LANG_BATCH | Byte3.REQ_DYN_BATCH | Byte3.REQ_GRID | Byte3.REQ_INSTID)),
+                ((byte)(Byte4.RPCPARAM_LOB | Byte4.DATA_USECS | Byte4.DATA_BIGDATETIME | Byte4.REQ_UNUSED4 | Byte4.REQ_UNUSED5 | Byte4.MULTI_REQUESTS | Byte4.REQ_MIGRATE | Byte4.REQ_UNUSED8)),
+                ((byte)(/*Byte5.REQ_DBRPC2 | */Byte5.REQ_CURINFO3 | Byte5.DATA_XML/* | Byte5.REQ_BLOB_NCHAR_16*/ | Byte5.REQ_LARGEIDENT/* | Byte5.DATA_SINT1 | Byte5.CAP_CLUSTERFAILOVER*/ | Byte5.DATA_UNITEXT)),
+                ((byte)(packetSizeRequestBit/* | Byte6.CSR_KEYSETDRIVEN*/ | Byte6.CSR_SEMISENSITIVE | Byte6.CSR_INSENSITIVE/* | Byte6.CSR_SENSITIVE*/ | Byte6.CSR_SCROLL | Byte6.DATA_INTERVAL | Byte6.DATA_TIME)),
+                ((byte)(Byte7.DATA_DATE | Byte7.BLOB_NCHAR_SCSU | Byte7.BLOB_NCHAR_8 | Byte7.BLOB_NCHAR_16 | Byte7.IMAGE_NCHAR | Byte7.DATA_NLBIN/* | Byte7.CUR_IMPLICIT*/ | Byte7.DATA_UINTN)),
+                ((byte)(Byte8.DATA_UINT8 | Byte8.DATA_UINT4 | Byte8.DATA_UINT2 | Byte8.REQ_RESERVED2 | Byte8.WIDETABLE | Byte8.DATA_COLUMNSTATUS | Byte8.OBJECT_BINARY | Byte8.REQ_RESERVED1)),
+                ((byte)(Byte9.OBJECT_CHAR/* | Byte9.OBJECT_JAVA1*/ | Byte9.DOL_BULK/* | Byte9.DATA_VOID*/ | Byte9.DATA_INT8 | Byte9.DATA_BITN | Byte9.DATA_FLTN | Byte9.PROTO_DYNPROC)),
+                ((byte)(/*Byte10.PROTO_DYNAMIC | */Byte10.DATA_BOUNDARY | Byte10.DATA_SENSITIVITY/* | Byte10.REQ_URGEVT | Byte10.PROTO_BULK*/ | Byte10.PROTO_TEXT/* | Byte10.CON_LOGICAL*/ | Byte10.CON_INBAND)),
+                ((byte)(/*Byte11.CON_OOB | */Byte11.CSR_MULTI/* | Byte11.CSR_REL | Byte11.CSR_ABS | Byte11.CSR_LAST | Byte11.CSR_FIRST | Byte11.CSR_PREV*/ | Byte11.DATA_MONEYN)),
+                ((byte)(Byte12.DATA_DATETIMEN | Byte12.DATA_INTN | Byte12.DATA_LBIN | Byte12.DATA_LCHAR | Byte12.DATA_DEC | Byte12.DATA_IMAGE | Byte12.DATA_TEXT | Byte12.DATA_NUM)),
+                ((byte)(Byte13.DATA_FLT8 | Byte13.DATA_FLT4 | Byte13.DATA_DATE4 | Byte13.DATA_DATE8 | Byte13.DATA_MNY4 | Byte13.DATA_MNY8 | Byte13.DATA_VBIN | Byte13.DATA_BIN)),
+                ((byte)(Byte14.DATA_VCHAR | Byte14.DATA_CHAR | Byte14.DATA_BIT | Byte14.DATA_INT4 | Byte14.DATA_INT2 | Byte14.DATA_INT1 | Byte14.REQ_PARAM | Byte14.REQ_MSG)),
+                ((byte)(Byte15.REQ_DYNF | Byte15.REQ_CURSOR/* | Byte15.REQ_BCP*/ | Byte15.REQ_MSTMT/* | Byte15.REQ_EVT*/ | Byte15.REQ_RPC | Byte15.REQ_LANG/* | Byte15.NONE*/)),
+
+                //cap response
+                ((byte)Byte16.Unknown),
+                ((byte)Byte17.Unknown),
+                ((byte)Byte18.Unknown),
```

Comment:
I think RES_UNUSED1 should be going here, seems to be misaligned compared to the previous file.

For TDS_CAP_RESPONSE the client tells the server what types it does not want to receive in responses, so most of the bits are 0 because the client's like "Yeah I'll consume it!"